README.md

```diff
@@ -1,24 +1,7 @@
 # README
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
-
-Things you may want to cover:
-
-* Ruby version
-
-* System dependencies
-
-* Configuration
-
-* Database creation
-
-* Database initialization
-
-* How to run the test suite
-
-* Services (job queues, cache servers, search engines, etc.)
-
-* Deployment instructions
-
-* ...
+FE do this for the DB:
+bundle install
+rails db:{drop,create,migrate,seed}
+rails s
+localhost:3001/graphiql
```

Comment:
Notes for FE to set up DB

Reply:
May need to add installing bundle with 'gem install bundler'

Reply:
These are the commands I ran:

```
sudo gem install bundler
brew install rbenv
echo 'eval "$(rbenv init -)"' >> ~/.bash_profile
source ~/.bash_profile
rbenv global 2.7.2
sudo bundle install
sudo gem install rails -v 5.2.6
sudo brew install postgresql
brew services start postgresql
rails db:{drop,create,migrate,seed}
rails s
```